schemas/schema.json

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
I know what schemas are used for generally, but when would this schema be used? I don't see us validating the structure of pings before sending them.

Reply:
It's not just for validating. It helps the analyst to know what exactly is contained in each ping and each field.